h2gis-functions/src/main/java/org/h2gis/functions/spatial/properties/ST_DistanceSphere.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * H2GIS is a library that brings spatial support to the H2 Database Engine
+ * <http://www.h2database.com>. H2GIS is developed by CNRS
+ * <http://www.cnrs.fr/>.
+ *
+ * This code is part of the H2GIS project. H2GIS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation;
+ * version 3.0 of the License.
+ *
+ * H2GIS is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details <http://www.gnu.org/licenses/>.
+ *
+ *
+ * For more information, please consult: <http://www.h2gis.org/>
+ * or contact directly: info_at_h2gis.org
+ */
+
+package org.h2gis.functions.spatial.properties;
+
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.geom.LineString;
+import com.vividsolutions.jts.geom.Point;
+import com.vividsolutions.jts.geom.Polygon;
+
+import org.h2gis.api.DeterministicScalarFunction;
+
+/**
+ * For geometry type returns minimum distance in meters between two geometries
+ * @author Michael MATUR
+ */
+public class ST_DistanceSphere extends DeterministicScalarFunction {
+
+    private static final double EARTH_RADIUS_METER = 6370986;
+
+    /**
+     * Default constructor
+     */
+    public ST_DistanceSphere() {
+        addProperty(PROP_REMARKS, "Returns minimum distance in meters between two lon/lat points. Uses a spherical earth and radius derived from the spheroid defined by the SRID");
```

Comment:
The document must be modified because the code doesn't check the SRID of the geometries but assumes its in WGS 84 long lat.